modules/ROOT/pages/environments.adoc

```diff
@@ -27,10 +27,13 @@ To create a new environment or manage an existing environment:
 
 . Sign into Anypoint Platform with an Organization Administrators role.
 . Go to Access Management.
-. From the left-hand menu, click *Environments*.
+. In the menu on the left, click *Environments*.
 . Click *Add Environment*.
 . Enter a name for the environment.
-. Select the environment type, then click *Create*.
+. Select the environment type.
+. Optionally, select at least one xref:managing-api-clients.adoc[client provider] for your environment.
+. Optionally, if you added more than one client provider, click *Set as default* to designate a default client provider.
```

Comment:
```suggestion
. Optionally, if you added more than one client provider, click *Set as default* to designate a default client provider.
```
Is this optional if you added more than one provider?

Reply:
Yes, technically, since a default option is chosen for them. They can change it if they so choose.

Reply:
```suggestion
. Optionally, if you added more than one client provider, click *Set as default* to designate a default client provider. If you do not select a default client provider, the first client provider that you added is the default.
```